File: code/src/test/scala/cern/colt/matrix/tint/impl/SparseRCMIntMatrix2DViewTest.scala
package cern.colt.matrix.tint.impl

//remove if not needed
import scala.collection.JavaConversions._

class SparseRCMIntMatrix2DViewTest(arg0: String) extends SparseRCMIntMatrix2DTest(arg0) {

  protected def createMatrices() {
    A = new SparseRCMIntMatrix2D(NCOLUMNS, NROWS).viewDice()
    B = new SparseRCMIntMatrix2D(NCOLUMNS, NROWS).viewDice()
    Bt = new SparseRCMIntMatrix2D(NROWS, NCOLUMNS).viewDice()
  }
}

File: src/test/java/cernx/colt/matrix/tint/impl/SparseRCMIntMatrix2DViewTest.java
package cernx.colt.matrix.tint.impl;

public class SparseRCMIntMatrix2DViewTest extends SparseRCMIntMatrix2DTest {

    public SparseRCMIntMatrix2DViewTest(String arg0) {
        super(arg0);
    }

    protected void createMatrices() throws Exception {
        A = new SparseRCMIntMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        B = new SparseRCMIntMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        Bt = new SparseRCMIntMatrix2D(NROWS, NCOLUMNS).viewTranspose();
    }

}
